src/components/scenes/Loans/LoanCreateConfirmationScene.tsx

```diff
@@ -107,7 +107,7 @@ export const LoanCreateConfirmationScene = (props: Props) => {
 
       // Target mainnet native balance should be double the fees estimate to be
       // extra generous when accounting for fee volatility.
-      const feeNativeAmount = sub(mul(mainNetworkFeeAmount, '2'), borrowWalletNativeBalance)
+      const feeNativeAmount = sub(mul(mainNetworkFeeAmount, '1.25'), borrowWalletNativeBalance)
```

Comment:
Swaps can take a very long time to complete. If we cover 2x we're ensuring the product works even if the fees double during the time we're waiting on swaps. If they don't double, then the user has a bit more ETH left over; so what, that can be used for future contract calls to manage their loan. Much better to overshoot and have extra ETH then to undershoot and fail the program asking the user to recover the broken loan state, don't you think?

Reply:
That's fair, I did not consider the possible length of time. However, if I didn't consider it, another user wouldn't either, and would be understandably confused about this.